README.md

```diff
@@ -16,82 +20,811 @@ yarn add gotrue-js
-  error => console.log("Failed to log in: %o", error)
-);
-// Use `auth.confirm(token, true)` to save to localStorage
-
-auth.login(email, password).then(
-  user => console.log("Logged in as %s", user.email),
-  error => console.log("Failed to log in: %o", error);
-)
-// Use `auth.login(email, password, true)` to save to localStorage
-
-auth.requestPasswordRecovery(email).then(
-  response => console.log("Recovery email sent"),
-  error => console.log("Error sending recovery mail: %o", error)
-);
-
-auth.recover(token).then(
-  user => console.log("Logged in as %s", user.email),
-  error => console.log("Failed to verify recover token: %o", error)
-);
-
-const user = auth.currentUser()
-
-// Users updating email requires an email confirmation step
-user.update({email: newEmail, password: newPassword}).then(
-  user => console.log("Updated user"),
-  error => console.log("Failed to update user: %o", error)
-);
-
-user.jwt().then(
-  token => console.log("Current token: %s", token),
-  error => console.log("Failed to get token: %o", error)
-);
-
-user.logout().then(
-  response => console.log("User logged out"),
-  error => console.log("Failed to logout user: %o", error)
-);
-
-const admin = user.admin // User must have the 'admin' role
-
-admin.list(audience).then(
-  users => console.log("List of users: %o", users),
-  error => console.log("Failed to get list of users: %o", error)
-)
-
-const user = { id: 1234 }
-admin.getUser(user).then( 
-  user => console.log("User object: %o", user),
-  error => console.log("Failed to retrieve user: %o", error)
-)
-
-// Admins updating email does not require a confirmation step
-const attributes = {email: newEmail, password: newPassword}
-admin.updateUser(user, attributes).then(
-  user => console.log("Updated user: %o", user),
-  error => console.log("Failed to update user: %o", error)
-)
-
-admin.createUser(email, password, attributes = {}).then(
-  user => console.log("Created user user: %o", user),
-  error => console.log("Failed to create user: %o", error)
-)
-
-admin.deleteUser(user).then(
-  user => console.log("Deleted user user: %o", user),
-  error => console.log("Failed to delete user: %o", error)
-)
+import GoTrue from "gotrue-js";
+
+// Instantiate the GoTrue auth client with an optional configuration
+
+auth = new GoTrue({
+  APIUrl: "https://<your domain name>/.netlify/identity",
+  audience: "",
+  setCookie: false
+});
+```
+
+### GoTrue configuration
+
+APIUrl: The absolute path of the GoTrue endpoint. To find the `APIUrl`, go to `Identity` page of your Netlify site dashboard.
+
+audience(optional): `audience` is one of the pre-defined [JWT payload](https://tools.ietf.org/html/rfc7519#section-4.1.3) claims. It's an optional attribute which is set to be empty by default. If you were hosting your own identity service and wanted to support [multitenancy](https://en.wikipedia.org/wiki/Multitenancy), you would need `audience` to separate the users.
```

Comment:
Great, didn't see this wrt to the above comment.  Still, this should be described in the gotrue repo and we should link to that, when its written.  I would say this is the best we can do now in this repo.

Reply:
I'll break this out into a separate issue.

Reply:
https://github.com/netlify/gotrue/issues/173